.pre-commit-config.yaml

```diff
@@ -2,22 +2,22 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.2.0
     hooks:
-      # - id: trailing-whitespace
-      # - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
       - id: check-yaml
       - id: debug-statements
-      # - id: double-quote-string-fixer
       # - id: name-tests-test
+      #   args: ["--django"]
```

Comment:
I'm leaving this for the future. For when I actually want to run test and want to make sure things are actually running.